modules/base/rotation/timelinerotation.h

```diff
@@ -25,6 +25,7 @@
 #ifndef __OPENSPACE_MODULE_BASE___TIMELINEROTATION___H__
 #define __OPENSPACE_MODULE_BASE___TIMELINEROTATION___H__
 
+#include <openspace/properties/scalar/boolproperty.h>
```

Comment:
Order should be:
```cpp
#include <openspace/scene/rotation.h>

#include <openspace/properties/scalar/boolproperty.h>
#include <openspace/util/timeline.h>
```

Since the rotation is the base class, it comes first in the list and separated by an empty line